public_html/wp-content/plugins/wcpt/views/common/metabox-log.php

```diff
@@ -15,10 +40,10 @@
 
 			<?php foreach ( $entries as $entry ) : ?>
 				<tr class="<?php echo esc_attr( str_replace( '_', '-', $entry['type'] ) ); ?>">
-					<th><p><?php echo esc_html( date( 'Y-m-d h:ia', $entry['timestamp'] ) ); ?></p></th>
+					<th><p><?php echo esc_html( gmdate( 'Y-m-d h:ia', $entry['timestamp'] ) ); ?></p></th>
 					<th><p><?php echo esc_html( ucwords( str_replace( '_', ' ', $entry['type'] ) ) ); ?></p></th>
 					<th><p><?php echo esc_html( $entry['user_display_name'] ); ?></p></th>
-					<th><?php echo wp_kses( wpautop( $entry['message'] ), wp_kses_allowed_html( 'data' ) ); ?></th>
+					<th><?php echo wp_kses( wpautop( $entry['message'] ), $allowed_html ); ?></th>
```

Comment:
Could you use [`make_clickable`](https://developer.wordpress.org/reference/functions/make_clickable/) here, so that you don't need to create a new regex above? It also handles more edge-cases (for example, I added punctuation to my link when testing, and it considered that part of the link, ex `https://wordpress.org.`)

Reply:
Wow, wasn't aware of that kind of function! Will definitely to change that to core way.